var/spack/repos/builtin/packages/elemental/package.py

```diff
@@ -145,7 +150,14 @@ def cmake_args(self):
             libfortran = LibraryList(mpif77('--print-file-name',
                                             'libgfortran.%s' % dso_suffix,
                                             output=str))
-        if libfortran:
+        elif self.spec.satisfies('%xl') or self.spec.satisfies('%xl_r'):
+            xl_fort = env['SPACK_F77']
+            xl_bin = os.path.dirname(xl_fort)
+            xl_root = os.path.dirname(xl_bin)
+            libfortran = LibraryList('{0}/lib/libxlf90_r.{1}.1'
+                                     .format(xl_root, dso_suffix))
+
+        if 'libfortran' in locals():
```

Comment:
IMO it would be slightly clearer if you added an `else: libfortran = None` above.

Reply:
Done